src/includes/set-release/dotnet.nlog.mdx

```diff
@@ -0,0 +1,50 @@
+<Alert level="" title="NLog Layouts">
+
+For more information on how to dynamically set event data via `NLog.config`, see NLog's [layout renderer documentation](https://nlog-project.org/config/?tab=layout-renderers).
+
+</Alert>
+
+```xml {tabTitle:NLog.config}
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+>
+  <extensions>
+    <add assembly="Sentry.NLog" />
+  </extensions>
+  <targets>
+    <target
+      xsi:type="Sentry"
+      name="sentry"
+      dsn="___PUBLIC_DSN___"
+      release="my-project-name@2.3.12">
+    </target>
+  </targets>
+
+  <rules>
+    <logger name="*" writeTo="sentry" />
+  </rules>
+</nlog>
+```
+
+```csharp
+using Sentry;
+
+// Add this to the SDK initialization callback
+options.Release = "my-project-name@2.3.12";
+```
+
+```fsharp
+open Sentry
+
+// Add this to the SDK initialization callback
+options.Release <- "my-project-name@2.3.12"
+```
+
+The SDK attempts to locate the release to add to events sent to Sentry.
+
+The SDK will first look at the [entry assembly's](<https://msdn.microsoft.com/en-us/library/system.reflection.assembly.getentryassembly(v=vs.110).aspx>) `AssemblyInformationalVersionAttribute`, which accepts a string as
+value and is often used to set a GIT commit hash.
+
+If that returns null, it'll look at the default `AssemblyVersionAttribute` which accepts the numeric version number. When creating a project with Visual Studio, the value is set to _1.0.0.0_.
+Since that usually means that the version is either not being set, or is set via a different method. The **automatic version detection will disregard** this value and no _Release_ will be reported automatically.
```

Comment:
We dropped this thing so now it'll report 1.0.0 too so at least something shows up in Sentry (better than the alternative that nothing showed up)

```suggestion
If that returns null, it'll look at the default `AssemblyVersionAttribute` which accepts the numeric version number.
```

Could you please fix it in the main .NET page too?